packages/signhub/src/signers/signer.ts

```diff
@@ -25,6 +25,11 @@ export interface SapientSigner {
    */
   sign(message: ethers.BytesLike, metadata: object): Promise<ethers.BytesLike>
 
+  /**
+   * Build a validation signature for an undeployed contract signer.
+   */
+  buildValidationSignature?(signatureBytes: ethers.BytesLike): Promise<ethers.BytesLike | undefined>;
```

Comment:
Is this a 6492 signature or something else? if so, can we add 6492 to the comment?

Reply:
It is for now but I guess it doesn't have to be. If we extend support to other validation encodings (zk or something?) we could reuse this method. 
Happy to rename if you think we won't reuse later.

Reply:
Let’s rename it for now. I’d prefer it to be obvious at a glance.